### PR TITLE
[ Frontend ] Resolves #176 : Creation of jobs bug

### LINKED ISSUE #176: [ Software ] Don't refuse to create a new job with a player in database
Actually we refuse with a popup to create a jobs affected to a player who already in database.
The improvement consisted to create this jobs but not the player because it is impossible :)

### PR DESCRIPTION
**Pull request by yannis-mlgrn**
- Fixes #176 
## Features : 
- Fixed bug, now we can crate multiples jobs linked to the same player ( [0b4730e](https://github.com/mdl29/donkeycarLPH/pull/177/commits/0b4730e46ee5aa7b097c89cc86d574847d0d19ce) )
-  improve submit button of register popup for phone clients ( [d71dafb](https://github.com/mdl29/donkeycarLPH/pull/177/commits/d71dafb84bb9b69a73a847e95a34c04c7d9cfe6d) )

### CODE REVIEW

**Comment by yannis-mlgrn** [comment]
LGTM

File: donkeycarManagerFront/src/components/register.vue [hunk 1]
```diff
@@ -81,31 +66,25 @@ export default {
     },
     async addUser () {
       this.playerTest = await srv.getPlayerByPseudo(this.pseudo)
-      console.log(this.playerTest)
-      console.log(this.playerTest.length)
       if (this.playerTest.length === 0){
-        console.log('pass')
         this.player = await srv.createPlayer(this.pseudo)
-        console.log('pass1.1')
         await srv.addJobs(this.player.player_id)
```
Comment (Benvii) [inline donkeycarManagerFront/src/components/register.vue:71]: This should be outside the condition, in both case you will create the job you have duplicate code.

File: donkeycarManagerFront/src/components/register.vue [hunk 2]
```diff
@@ -81,31 +66,25 @@ export default {
...
-        } else if (this.waitingList.length >= 4 && this.waitingList.length < 8) {
-          this.attente = '30 minutes'
-        } else if (this.waitingList.length >= 8 && this.waitingList.length < 12) {
-          this.attente = '45 minutes'
-        } else if (this.waitingList.length >= 12) {
-          this.attente = 'plus de 1 heure'
-        }
         this.numero = this.player.player_id
+      } else if (this.playerTest.length !== 0){
+        await srv.addJobs(this.playerTest[0].player_id)
```
Comment (Benvii) [inline donkeycarManagerFront/src/components/register.vue:74]: Duplicated, put it outside the conditional block will be cleaner, less lines => less bugs

**Review by Benvii — CHANGES_REQUESTED** [review]
Small change required, conditional optimisation.

Algo : 

- Fetch player as this.player
- If not existing: create it;  set this.player
- Create the job (not in a conditional block)

**Comment by viandoxdev** [comment]
Closing for now since #180 should already fix that.

**Pull request closed without merging**